packages/askui-nodejs/src/utils/transformations.ts

```diff
@@ -40,21 +40,26 @@ export async function resizeBase64ImageWithSameRatio(
   }
   try {
     let resizeRatio = 1;
-    const orignalImage = await read(Buffer.from(base64ImageString.replace(/^data:image\/png;base64,/, ''), 'base64'));
-    const imageHeight = orignalImage.getHeight();
-    const imageWidth = orignalImage.getWidth();
+    const orignalImage = sharp(Buffer.from(base64ImageString.replace(/^data:image\/png;base64,/, ''), 'base64'));
+    const imageMetadata = await orignalImage.metadata();
+    const imageHeight = imageMetadata.height;
+    const imageWidth = imageMetadata.width;
 
-    if (Math.max(imageHeight, imageWidth) <= maxEdge) {
+    if (Math.max(imageHeight as number, imageWidth as number) <= maxEdge) {
```

Comment:
Do we really need the typecasts? If yes, they should be done as early as possible, i.e., on assignment of variables. This also allow multiple typecasts, e.g., in line 52 or 61.